### PR TITLE
Pim 4187 timeout

### PR DESCRIPTION
**Pull request by sumbobyboys**

### CODE REVIEW

File: features/Context/Page/Product/Edit.php [hunk 1]
```diff
@@ -360,36 +373,49 @@ protected function getFieldType($label)
      *
      * @return NodeElement
      */
-    protected function findCompoundField($name, $subLabelText)
+    protected function findCompoundField($name, $value, $currency)
```
Comment (sumbobyboys) [inline features/Context/Page/Product/Edit.php]: mybe rename this method as it's only used for prices

File: features/product/add_new_option_to_choice_attribute.feature [hunk 2]
```diff
@@ -23,10 +24,10 @@ Feature: Add a new option to a choice attribute directly from the product edit f
     Then the product Size should be "47 (XXL)"
 
   Scenario: Sucessfully add a new option to a multi select attribute
-    Given I add a new option to the "Weather conditions" attribute
+    Given I add a new option to the "weather_conditions" attribute
```
Comment (sumbobyboys) [inline features/product/add_new_option_to_choice_attribute.feature]: we should use the label of the attribute

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/multi-select-field.js [hunk 3]
```diff
@@ -58,13 +58,15 @@ define(
                             }
                         },
                         initSelection: function (element, callback) {
-                            var choices = _.map($(element).val().split(','), function (choice) {
-                                return {
-                                    id: choice,
-                                    text: choice
-                                };
+                            debugger;
```
Comment (sumbobyboys) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/multi-select-field.js]: please remove this one

File: features/Context/Page/Product/Edit.php (hunk 1 above)
Comment (jmleroux) [inline features/Context/Page/Product/Edit.php]: A `findPriceField` already exists in the base form.

**Pull request merged**